R.ODAF.utils/R/prepare_data.R

```diff
@@ -48,13 +50,16 @@ prepare_data_case2 <- function(params,
                                mergedDEGsList,
                                exp_metadata,
                                designList,
-                               contrasts) {
+                               contrasts,
+                               biomarkers) {
    message("Prepare Data for Case 2: No DESeq2 Facet, Yes Report Facet")
-   display_group_filter <- params$reports_filter
+   reports_filter <- params$reports_filter
```

Comment:
Interesting, I kind of remember this, were there any other implications? Did you have to make this change anywhere else?

Reply:
I think this is just a change you forgot to make for case 2 while doing the big R.ODAF.utils update. I didn't have to change it anywhere else.

Reply:
I guess it failed when you tried to run that condition but it never came up because it wasn't tested before. I remember now